docs/content-services/components/search-date-range-advanced-tabbed.component.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: Search date range advanced component
+Added: v6.1.0
```

Comment:
This is no longer correct version if you're adding this component in Aquarius

Reply:
Updated version to 6.2.0